packages/core/src/import-runs.ts

```diff
@@ -4,7 +4,7 @@ import { RunPage } from "./types";
 
 export async function importRunsFromPath(globPath: string, dir: string): Promise<RunPage[]> {
   const files = await glob(globPath);
-  const promises = files.map(f => resolve(dir, "../", f));
```

Comment:
this is a breaking change I need to figure out

Reply:
Resolved, ended up using `process.cwd()` and it worked flawlessly. `dir` option has been marked deprecated